frontend/src/components/eMIB/constants.js

```diff
@@ -44,7 +44,14 @@ export const actionShape = PropTypes.shape({
   reasonsForAction: PropTypes.string,
   task: PropTypes.string,
   emailType: PropTypes.oneOf(Object.keys(EMAIL_TYPE)),
-  emailTo: PropTypes.string,
-  emailCc: PropTypes.string,
+  emailTo: PropTypes.array,
```

Comment:
should this be `PropTypes.arrayOf(contactShape)`? or is it an array of strings, either way, can you specify?

Reply:
fixed in 320932d